das_client/lib/app/pages/journey/train_journey/widgets/table/service_point_row.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:das_client/app/widgets/table/das_table_cell.dart';
+import 'package:das_client/app/widgets/table/das_table_row.dart';
+import 'package:das_client/sfera/sfera_component.dart';
+import 'package:design_system_flutter/design_system_flutter.dart';
+import 'package:flutter/material.dart';
+
+// TODO: Extract real values from SFERA objects.
+@immutable
+class ServicePointRow extends DASTableRowBuilder {
+  const ServicePointRow({
+    this.timingPoint,
+    this.timingPointConstraints,
+    this.active = false,
+    super.height = 64.0,
+  });
+
+  final TimingPoint? timingPoint;
+  final TimingPointConstraints? timingPointConstraints;
+  final bool active;
+
+  final Alignment _defaultRowAlignment = Alignment.bottomCenter;
+
+  @override
+  DASTableRow build(BuildContext context) {
+    return DASTableRow(
+      height: height,
+      color: active ? SBBColors.royal.withOpacity(0.2) : null,
+      cells: [
+        _kilometre(),
```

Comment:
könnte man diesen part nicht in den BaseBuilder schieben, dass nicht jede Row diesen Aufbau neu machen muss, sondern einfach die einzelnen Cells die benötigt werden überschreiben? 
KM wird ja z.b. immer quasi gleich sein, da gibts kein Grund das immer neu zu schreiben. 

Auch wäre es dann leichter einzelne Spalten auszublenden, was in einem späteren Feature noch kommt.